src/Symfony/Component/DomCrawler/Tests/Html5ParserCrawlerTest.php

```diff
@@ -60,6 +60,30 @@ public function testHtml5ParserNotSameAsNativeParserForSpecificHtml()
         $this->assertNotEquals($nativeCrawler->filterXPath('//h1')->text(), $html5Crawler->filterXPath('//h1')->text(), 'Native parser and Html5 parser must be different');
     }
```

Comment:
Shouldn't this prior test (`testHtml5ParserNotSameAsNativeParserForSpecificHtml`) be removed now?

Reply:
Hmm, that's not a perfect test but it still validating a certain behaviour of the domCrawler component and can be interesting to keep ?

Reply:
Yes, this test case is useful to me, it tests the behavior and that's what we need also.
If the behavior of the xml parser changes, we'll know also, and this is good.